src/components/TimeInput/index.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { _cs } from '@togglecorp/fujs';
+
+import useUiModeClassName from '../../hooks/useUiModeClassName';
+import InputContainer, { Props as InputContainerProps } from '../InputContainer';
+import RawInput, { Props as RawInputProps } from '../RawInput';
+
+import styles from './styles.css';
+
+type InheritedProps<T> = (Omit<InputContainerProps, 'input'> & RawInputProps<T>);
+export interface Props<T extends string> extends InheritedProps<T> {
+    inputElementRef?: React.RefObject<HTMLInputElement>;
+    inputClassName?: string;
+}
+
+function TimeInput<T extends string>(props: Props<T>) {
```

Comment:
@frozenhelium is there a way we can set 24 hour or 12 hour format?

Reply:
Nope